notebooks/working/ho_deconfounder_on_sim_data.py

```diff
@@ -0,0 +1,512 @@
+                          'ASC Walk',
+                          'ASC Bike']
+
+mnl_specification_noncausal["total_travel_time"] = [1, 2, 3, [4, 5, 6]]
+mnl_names_noncausal["total_travel_time"] = ['Travel Time, units:min (Drive Alone)',
+                                  'Travel Time, units:min (SharedRide-2)',
+                                  'Travel Time, units:min (SharedRide-3+)',
+                                  'Travel Time, units:min (All Transit Modes)']
+
+mnl_specification_noncausal["total_travel_cost"] = [1, 2, 3, [4, 5, 6]]
+mnl_names_noncausal["total_travel_cost"] = ['Travel Cost, units:$ (Drive Alone)',
+                                  'Travel Cost, units:$ (SharedRide-2)',
+                                  'Travel Cost, units:$ (SharedRide-3+)',
+                                  'Travel Cost, units:$ (All Transit Modes)']
+
+# mnl_specification["cost_per_distance"] = [1, 2, 3]
+# mnl_names["cost_per_distance"] = ["Travel Cost per Distance, units:$/mi (Drive Alone)",
+#                                   "Travel Cost per Distance, units:$/mi (SharedRide-2)",
+#                                   "Travel Cost per Distance, units:$/mi (SharedRide-3+)"]
+
+mnl_specification_noncausal["cars_per_licensed_drivers"] = [[1, 2, 3]]
+mnl_names_noncausal["cars_per_licensed_drivers"] = ["Autos per licensed drivers (All Auto Modes)"]
+
+# mnl_specification["total_travel_distance"] = [1, 2, 3, 7, 8]
+# mnl_names["total_travel_distance"] = ['Travel Distance, units:mi (Drive Alone)',
+#                                       'Travel Distance, units:mi (SharedRide-2)',
+#                                       'Travel Distance, units:mi (SharedRide-3+)',
+#                                       'Travel Distance, units:mi (Walk)',
+#                                       'Travel Distance, units:mi (Bike)']
+
+# mnl_specification["cross_bay"] = [[2, 3], [4, 5, 6]]
+# mnl_names["cross_bay"] = ["Cross-Bay Tour (Shared Ride 2 & 3+)",
+#                           "Cross-Bay Tour (All Transit Modes)"]
+mnl_specification_noncausal["cross_bay"] = [[2, 3]]
+mnl_names_noncausal["cross_bay"] = ["Cross-Bay Tour (Shared Ride 2 & 3+)"]
+
+mnl_specification_noncausal["household_size"] = [[2, 3]]
+mnl_names_noncausal["household_size"] = ['Household Size (Shared Ride 2 & 3+)']
+
+mnl_specification_noncausal["num_kids"] = [[2, 3]]
+mnl_names_noncausal["num_kids"] = ["Number of Kids in Household (Shared Ride 2 & 3+)"]
+
+# +
+# Estimate the basic MNL model, using the hessian and newton-conjugate gradient
+mnl_model_noncausal = cm.create_choice_model(data=data,
+                                   alt_id_col="mode_id",
+                                   obs_id_col="observation_id",
+                                   choice_col="sim_choice",
+                                   specification=mnl_specification_noncausal,
+                                   model_type="MNL",
+                                   names=mnl_names_noncausal)
```

Comment:
If there's no reason otherwise, I suggest following PEP8 [indentation guidelines](https://www.python.org/dev/peps/pep-0008/#indentation) by lining up all the arguments with `data=data`.